<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="DESKTOP-M7577V9" failures="1" tests="5" name="Test" time="0.065" errors="0" timestamp="2022-09-06T14:42:34 WEST">
  <testcase classname="test.Day2" name="loginAfiliates" time="0.005"/>
  <testcase classname="test.Day1" name="demo" time="0.006"/>
  <testcase classname="test.Day2" name="loginClients" time="0.001"/>
  <testcase classname="test.Day2" name="loginWorkers" time="0.0"/>
  <testcase classname="test.Day1" name="demo2" time="0.007">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at test.Day1.demo2(Day1.java:22)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base/java.lang.Thread.run(Thread.java:833)
... Removed 16 stack frames]]>
    </failure>
  </testcase> <!-- demo2 -->
  <testcase classname="test.Day2" name="ugaBudsass" time="0.0">
    <ignored/>
  </testcase> <!-- ugaBudsass -->
  <testcase classname="test.Day2" name="ugaBudsa" time="0.0">
    <ignored/>
  </testcase> <!-- ugaBudsa -->
</testsuite> <!-- Test -->
